target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1210-12.dts

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "rtl930x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "zyxel,xgs1210-12", "realtek,rtl838x-soc";
+	model = "Zyxel XGS1210-12 Switch";
+
+	// mw.l 0xb8003308 0x00200000 active low toggles PHY reset
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	i2c0: i2c-rtl9300 {
+		compatible = "realtek,rtl9300-i2c";
+		reg = <0x1b00036c 0x3c>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		sda-pin = <9>;
+		scl-pin = <8>;
+	};
+
+	i2cmux {
+		compatible = "realtek,i2c-mux-rtl9300";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		i2c-parent = <&i2c0>;
+
+		/* i2c of the left SFP cage: port 11 */
```

Comment:
These comments tend to be tricky, the left on the front, or the left as seen from the rear when looking from the SOC/PCB PoV?
```suggestion
 		/* i2c of the left SFP+ cage as seen from the front: port 11 */ 
```

Reply:
Will be updated as suggested.

Reply:
Will be fixed as suggested.